scripts/build-for-testing.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Build the quickstart. If we're running on the main repo (not a fork), we
```

Comment:
Add a comment about the script that this was modified from - to make it easier if we want to combine in the future.